src/Sql/Ddl/CreateTable.php

```diff
@@ -146,10 +113,9 @@ protected function processTable(PlatformInterface $adapterPlatform = null)
 
     /**
      * @param PlatformInterface $adapterPlatform
-     *
-     * @return string[][]|null
+     * @return string[][]|void
```

Comment:
I think we are trying to avoid that, and do not mix void with the other type

Reply:
changed